scripts/svm/bridgeLiabilityToHubPool.ts

```diff
@@ -0,0 +1,305 @@
+    svmSpokeProgram.programId
+  );
+  const tokenMessengerMinterProgram = new Program<TokenMessengerMinter>(tokenMessengerMinterIdl, provider);
+
+  const [tokenMessengerMinterSenderAuthority] = PublicKey.findProgramAddressSync(
+    [Buffer.from("sender_authority")],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const [messageTransmitter] = PublicKey.findProgramAddressSync(
+    [Buffer.from("message_transmitter")],
+    messageTransmitterProgram.programId
+  );
+
+  const [tokenMessenger] = PublicKey.findProgramAddressSync(
+    [Buffer.from("token_messenger")],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const [remoteTokenMessenger] = PublicKey.findProgramAddressSync(
+    [Buffer.from("remote_token_messenger"), Buffer.from(anchor.utils.bytes.utf8.encode(ethereumDomain.toString()))],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const [tokenMinter] = PublicKey.findProgramAddressSync(
+    [Buffer.from("token_minter")],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const [localToken] = PublicKey.findProgramAddressSync(
+    [Buffer.from("local_token"), inputToken.toBuffer()],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const [cctpEventAuthority] = PublicKey.findProgramAddressSync(
+    [Buffer.from("__event_authority")],
+    tokenMessengerMinterProgram.programId
+  );
+
+  const messageSentEventData = anchor.web3.Keypair.generate(); // This will hold the message sent event data.
+  const bridgeTokensToHubPoolAccounts = {
+    payer: signer.publicKey,
+    mint: inputToken,
+    state: statePda,
+    transferLiability,
+    vault,
+    tokenMessengerMinterSenderAuthority,
+    messageTransmitter,
+    tokenMessenger,
+    remoteTokenMessenger,
+    tokenMinter,
+    localToken,
+    messageSentEventData: messageSentEventData.publicKey,
+    messageTransmitterProgram: messageTransmitterProgram.programId,
+    tokenMessengerMinterProgram: tokenMessengerMinterProgram.programId,
+    tokenProgram: TOKEN_PROGRAM_ID,
+    systemProgram: SystemProgram.programId,
+    cctpEventAuthority: cctpEventAuthority,
+    program: svmSpokeProgram.programId,
+  };
+
+  const initialVaultBalance = (await provider.connection.getTokenAccountBalance(vault)).value.amount;
+
+  const tx = await svmSpokeProgram.methods
+    .bridgeTokensToHubPool(new BN(amount))
+    .accounts(bridgeTokensToHubPoolAccounts)
+    .signers([messageSentEventData])
+    .rpc();
+
+  const finalVaultBalance = (await provider.connection.getTokenAccountBalance(vault)).value.amount;
+
+  console.log(`SVM Spoke Pool Initial Vault balance: ${formatUsdc(BigNumber.from(initialVaultBalance))} USDC.`);
+  console.log(`SVM Spoke Pool Final Vault balance: ${formatUsdc(BigNumber.from(finalVaultBalance))} USDC.`);
+  console.log(
+    `Sent ${formatUsdc(BigNumber.from(initialVaultBalance).sub(BigNumber.from(finalVaultBalance)))} USDC through CCTP.`
+  );
+
+  return tx;
+}
+
+// Run the bridgeLiabilityToHubPool function
+bridgeLiabilityToHubPool();
```

Comment:
this script is SUPER useful. This will basically be what the finalizer needs to do in production.